src/vehicle.cpp

```diff
@@ -3955,6 +3959,12 @@ void vehicle::idle(bool on_map) {
         engine_on = false;
     }
 
+    if ( planter_on && g->get_temperature() < 50) { // copied from iexamine::dirtmound
+        add_msg(m_info, _("Planter system stopped due to cold"));
```

Comment:
Should mention the vehicle name and possibly also check for visibility. `idle` is processed even when player doesn't see the vehicle.